Earthfile

```diff
@@ -306,17 +306,17 @@ base-image:
     END
 
     # TEST KAIROS-AGENT FROM BRANCH
-    ARG KAIROS_AGENT_DEV
-    ARG KAIROS_AGENT_DEV_BRANCH=main
+    ARG KAIROS_AGENT_DEV=true
+    ARG KAIROS_AGENT_DEV_BRANCH=1448-use-yip
     IF [ "$KAIROS_AGENT_DEV" = "true" ]
         RUN rm -rf /usr/bin/kairos-agent
         COPY github.com/kairos-io/kairos-agent:$KAIROS_AGENT_DEV_BRANCH+build-kairos-agent/kairos-agent /usr/bin/kairos-agent
     END
     # END
 
     # TEST IMMUCORE FROM BRANCH
-    ARG IMMUCORE_DEV
-    ARG IMMUCORE_DEV_BRANCH=master
+    ARG IMMUCORE_DEV=true
+    ARG IMMUCORE_DEV_BRANCH=1448-use-yip
     IF [ "$IMMUCORE_DEV" = "true" ]
```

Comment:
This PR is not meant to be merged as is. It's pointing to dev branches of kairos-agent and immucore which consume this yip version: https://github.com/mudler/yip/pull/96

The merging process should be:
- merge the yip PR
- bump kairos-agent and immucore to use that yip version
- tag new releases of kairos-agent and immucore
- consume those versions in kairos

The rest of the changes in this PR can be then merged or moved to another PR.